datahub-graphql-core/src/main/java/com/linkedin/datahub/graphql/types/mlmodel/mappers/MLFeatureTablePropertiesMapper.java

```diff
@@ -41,6 +42,8 @@ public MLFeatureTableProperties apply(@NonNull final com.linkedin.ml.metadata.ML
                 .collect(Collectors.toList()));
         }
 
+        result.setCustomProperties(StringMapMapper.map(mlFeatureTableProperties.getCustomProperties()));
```

Comment:
@maaaikoool I think we need a `(if mlFeatureTableProperties. getCustomProperties() != null)` check here

Reply:
I believe that `customProperties` is non nullable ( it has a `@Nonnull` annotation )

Reply:
Yep, it is not optional https://github.com/datahub-project/datahub/blob/master/metadata-models/src/main/pegasus/com/linkedin/common/CustomProperties.pdl#L15

Reply:
I've double checked with other mappers and it seems that indeed we are using a conditional assignment checking for `hasCustomProperties` so I've added it 👌